src/cpp/include/openvino/genai/image_generation/flux_transformer_2d_model.hpp

```diff
@@ -0,0 +1,66 @@
+// Copyright (C) 2023-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include <filesystem>
+#include <string>
+
+#include "openvino/core/any.hpp"
+#include "openvino/runtime/infer_request.hpp"
+#include "openvino/runtime/properties.hpp"
+#include "openvino/runtime/tensor.hpp"
+
+#include "openvino/genai/visibility.hpp"
+
+namespace ov {
+namespace genai {
+
+class OPENVINO_GENAI_EXPORTS FluxTransformer2DModel {
+public:
+    struct Config {
+        size_t in_channels = 64;
+
+        std::vector<size_t> block_out_channels = { 128, 256, 512, 512 };
```

Comment:
looks like later we need to add `guidance_embeds`, as other Flux models rely on this parameter https://huggingface.co/Freepik/flux.1-lite-8B-alpha/blob/main/transformer/config.json#L11 and embed guidance scale

Reply:
Yes, but currently I don't have `guidance` input for IR